spdx/package.py

```diff
@@ -224,7 +239,9 @@ def validate_mandatory_str_fields(self, messages):
         """Fields marked as Mandatory and of type string in class
         docstring must be of a type that provides __str__ method.
         """
-        FIELDS = ["name", "spdx_id", "download_location", "verif_code", "cr_text"]
+        FIELDS = ["name", "spdx_id", "download_location", "cr_text"]
+        if self.files_analyzed != False:
```

Comment:
```suggestion
        if not self.files_analyzed:
```

Reply:
This is on purpose. Specs say: 

> ## 3.8 Files Analyzed 
>
> [...]
>
> 3.8.3 Cardinality: Optional, one. **If omitted, the default value of true is assumed**.

So if files_analyzed is None, we have to assume that its value is True and do the check

Reply:
```suggestion
        if self.files_analyzed != False:
```

Reply:
```suggestion
        if self.files_analyzed != False:
            # FIXME: for clarity and better readability, self.files_analyzed should 
            # default to True when the corresponding value is omitted in the SPDX file
            # (see SPDX specs 3.8.3)
```

Reply:
is it OK if we put a reminder here, and then I will make another PR?